pkgs/default.nix

```
@@ -55,6 +55,7 @@
   mcp-server-time = pkgs.callPackage ./reference/time.nix { };
 
   # official servers
+  mcp-context7 = pkgs.callPackage ./official/context7 { };
```

```suggestion
  context7-mcp = pkgs.callPackage ./official/context7 { };
```
I'd like to use the same name as the npm package.